src/Core/Resolvers/SqlQueryEngine.cs

```diff
@@ -156,6 +156,23 @@ public async Task<IActionResult> ExecuteAsync(FindRequestContext context, string
                                        FormatFindResult(queryJson, context);
         }
 
+        /// <summary>
+        /// Given the FindRequestContext, obtains the query text, executes it against the backend and returns the result as a JsonDocument.
```

Comment:
nit: does it really obtain the query text here?

Reply:
Yeah. The `ExecuteAsync` method that is called builds the exact database query and then executes it against the database. The key difference between this and the existing method is that the existing one performs formatting so as to bring it to a state that can be returned in the response. But, this method returns the result as a `JsonDocument` which can be used for further processing.


I've noted this difference in the method summary